### PR TITLE
Upgrade icon library to Font Awesome 5

### PR DESCRIPTION
**Pull request by aubreyarcangel**
## Description
Updated styles, classnames, font library variable names, and dependances to support the icon library upgrade from Font Awesome 4 to Font Awesome 5 in the Design System. 

**Note: This PR cannot be merged until the design system ticket upgrade is merged first. https://github.com/department-of-veterans-affairs/design-system/pull/220

## Testing done
Visual testing on chrome browers

## Screenshots
Upgraded Icons in the home page benefits rows
screen shot 2019-01-11 at 3 44 20 pm

Updated icon-large on hub page
screen shot 2019-01-11 at 3 48 39 pm

Before:
screen shot 2019-01-11 at 3 52 39 pm

After:
screen shot 2019-01-11 at 3 54 24 pm

## Acceptance criteria
- [X] All new Font Awesome 5 icons appear.

## Definition of done
- [X] Events are logged appropriately
- [X] Documentation has been updated, if applicable
- [X] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by ncksllvn** [comment]
@amycesal @emilywaggoner When you all have time, can you do a design sweep through the review instance in this PR and make sure we didn't miss any icons?

File: src/platform/site-wide/sass/style-consolidated.scss [hunk 1]
```diff
@@ -30,10 +30,10 @@ $modal-layer: 400;
...
 
 @font-face {
-  font-family: "FontAwesome";
-  src: url("~font-awesome/fonts/fontawesome-webfont.woff2") format("woff2"),
-    url("~font-awesome/fonts/fontawesome-webfont.woff") format("woff"),
-    url("~font-awesome/fonts/fontawesome-webfont.eot") format("eot");
+  font-family: "Font Awesome 5 Free";
+  src: url("~@fortawesome/fontawesome-free/webfonts/fa-solid-900.woff2") format("woff2"),
+    url("~@fortawesome/fontawesome-free/webfonts/fa-solid-900.woff") format("woff"),
+    url("~@fortawesome/fontawesome-free/webfonts/fa-solid-900.eot") format("eot");
```
Comment (ncksllvn) [inline src/platform/site-wide/sass/style-consolidated.scss]: Can we update this to import fonts from the Formation module instead? If we can do this, then I think we can uninstall Font Awesome from `vets-website` altogether, and instead rely entirely on Formation's FA.

File: package.json [hunk 2]
```diff
@@ -183,7 +184,7 @@
   },
   "private": true,
   "dependencies": {
-    "@department-of-veterans-affairs/formation": "^1.18.0",
+    "@department-of-veterans-affairs/formation": "^2.0.0-fa",
```
Comment (ncksllvn) [inline package.json]: Just noting that this is a temporary version and will point to `2.0.0` once we're ready.

**Review by ncksllvn — COMMENTED** [review]
This all looks good to me! I can't approve until we've published the official Formation version, but we can do that soon.

File: package.json (hunk 2 above)
Comment (ncksllvn) [reply package.json]: Looks like we still need to uninstall the old Font Awesome, https://github.com/department-of-veterans-affairs/vets-website/pull/9369/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L203.

Can we also move `@fortawesome/fontawesome-free` down to `dependencies`? Right now it's listed under `devDependencies`. The distinction is sort of unclear, but we try to follow this general rule - 

> The difference between these two, is that devDependencies are modules which are only required during development, while dependencies are modules which are also required at runtime.

[article](https://medium.com/@dylanavery720/npmmmm-1-dev-dependencies-dependencies-8931c2583b0c)

**Review by emilywaggoner — APPROVED** [review]
This all looks good to me, @ncksllvn and @aubreyarcangel.

File: package.json (hunk 2 above)
Comment (aubreyarcangel) [reply package.json]: I removed Font Awesome 4 from `dependencies` and moved Font Awesome 5 from `devDependencies` to `dependencies`

**Pull request merged**